service/resource/cloudformation/error.go

```diff
@@ -31,3 +31,5 @@ func IsStackNotFound(err error) bool {
 	}
 	return strings.Contains(microerror.Cause(err).Error(), "does not exist")
 }
+
+var wrongTypeError = microerror.New("wrong type")
```

Comment:
I would like to have a matcher for this as well even if it is not yet used.

Reply:
Done.